lms/djangoapps/course_api/blocks/transformers/tests/test_milestones.py

```diff
@@ -66,14 +68,24 @@ def setup_proctored_exam(self, block, attempt_status, user_id):
         create_exam_attempt(exam_id, user_id, taking_as_proctored=True)
         update_attempt_status(exam_id, user_id, attempt_status)
 
+    def setup_gated_section(self, gated_block, gating_block):
+        """
+        Test helper to create a gating requirement.
+        Args:
+            gated_block: The block that should be inaccessible until gating_block is completed
+            gating_block: The block that must be completed before access is granted
+        """
+        gating_api.add_prerequisite(self.course.id, unicode(gating_block.location))
+        gating_api.set_required_content(self.course.id, gated_block.location, gating_block.location, 100)
+
     ALL_BLOCKS = ('course', 'A', 'B', 'C', 'TimedExam', 'D', 'E', 'PracticeExam', 'F', 'G')
 
     def get_course_hierarchy(self):
         """
         Get a course hierarchy to test with.
         """
 
-        #                  course
+        #                    course
```

Comment:
Super minor nit, more a question really - why was this spacing changed? It seems like it fit better before.